internal/uplink/gpustats.go

```diff
@@ -41,4 +41,5 @@ type GPUProcInfo struct {
 	Pid     uint64  `json:"pid"`
 	Name    string  `json:"name"`
 	MemUsed float64 `json:"used_memory"`
+	Owner   string  `json:"owner"`
```

Comment:
NIT: It's worth adding a comment here about `Owner` being the name of the user running the job, in human readable form.